### ISSUE TITLE
getFeatureImportance for models after feature selection?

### ISSUE THREAD

**Issue by slzhao** [mlr-org/mlr#1990]
```r
lrn = makeFilterWrapper(learner = "classif.rpart", fw.method = "information.gain", fw.abs = 2)
rdesc = makeResampleDesc("CV", iters = 10)
r = resample(learner = lrn, task = iris.task, resampling = rdesc, show.info = FALSE, models = TRUE)

r$aggr
sfeats = sapply(r$models, getFilteredFeatures)
table(sfeats)

getFeatureImportance(r$models[[1]])
#Error in UseMethod("getFeatureImportanceLearner") :
#no applicable method for 'getFeatureImportanceLearner' applied to an object of class "c('FilterWrapper', 'BaseWrapper', 'Learner')"
```

Hello,

I didn't find any introduction about the usage of getFeatureImportance function after feature selection. The example code and error message is listed above. Is it possible to do it? Thanks!

**Comment by ja-thomas** [comment]
Thanks for reporting, we don't have a method for this. Fixing this now. 

The same problem appears for `getOOBPreds`

**Comment by slzhao** [comment]
Thanks! It is great to know it is in development.

**Comment by berndbischl** [comment]
fixed in #1994
